src/wazuh_db/wdb.h

```diff
@@ -2060,6 +2061,25 @@ wdbc_result wdb_global_assign_agent_group(wdb_t *wdb, int id, cJSON* j_groups, i
  */
 wdbc_result wdb_global_unassign_agent_group(wdb_t *wdb, int id, cJSON* j_groups);
 
+/**
+ * @brief Return the number of groups are assigned to agent_id
```

Comment:
I propose something like: `Returns the number of groups that are assigned to an agent`

Reply:
Changed!